### ISSUE TITLE
PC often crashes during Sim. 

### ISSUE THREAD

**Issue by Traker2** [simulationcraft/simc#3392]
My PC often crashes during a simulation. It had seemed that closing all other programs before running the Sim would help ensure its completion without crashing. Today I have tried this multiple times and my PC crashes on each attempt. This has also happened with previous SimC versions. 
  #### Simulationcraft Version
715_01
#### If Simulationcraft is crashing, please include your dxdiag file - Instructions here - https://us.battle.net/support/en/article/Obtaining-System-Files
[DxDiag.txt](https://github.com/simulationcraft/simc/files/706327/DxDiag.txt)

#### simc_gui.simc file or contents
This file is saved in the same folder as the program after each simulation. Make sure to run the simulation that is giving you issues, then include the file here by either uploading to github, copy/pasting the contents, or using http://pastebin.com/
### Begin GUI options ###
item_db_source=local/bcpapi/wowhead
target_error=0
iterations=10000
default_world_lag=0.1
max_time=300
vary_combat_length=0.2
fight_style=Patchwerk
tmi_window_global=6
target_level+=3
target_race=Humanoid
default_skill=1.0
optimal_raid=0
override.bloodlust=1
override.bleeding=1
override.mortal_wounds=1
deterministic=1
threads=8
process_priority=Normal
calculate_scale_factors=1
scale_only=int,sp,crit,haste,mastery,vers
statistics_level=1
### Begin simulateText ###
rogue="Morthaas"
level=110
race=blood_elf
region=us
server=earthen_ring
role=
professions=leatherworking=311/skinning=800
talents=1133211
spec=assassination
artifact=43:0:0:0:0:323:3:324:3:325:3:328:3:330:3:331:3:332:1:334:1:337:1:346:1:347:1:1276:1

head=,id=139281,bonus_id=1727/1497/3336
neck=,id=132444,enchant_id=5439,bonus_id=3459/3458,gem_id=130222
shoulder=,id=134330,bonus_id=3473/1808/41/1502/1674
back=,id=141542,bonus_id=3466/1472
chest=,id=141487,bonus_id=40/3466/1477/3336
wrist=,id=139209,bonus_id=1807/1472
hands=,id=134239,bonus_id=3474/1808/1507/1674,gem_id=130215
waist=,id=134331,bonus_id=3473/1502/1674
legs=,id=139201,bonus_id=1807/1482/3336
feet=,id=134193,bonus_id=1727/40/1502/1813
finger1=,id=133679,bonus_id=1727/1808/1497/3336,gem_id=130215
finger2=,id=134245,bonus_id=3474/1808/1507/1674
trinket1=,id=121310,bonus_id=3473/605/1502/1674
trinket2=,id=134203,bonus_id=3432/603/1507/3336
main_hand=,id=128870,bonus_id=741,relic_id=1807:1477:3336/3470:1472:3336/1726:1497:3337,gem_id=138226/142511/136718/0
off_hand=,id=128869,relic_id=0/0,gem_id=0/0/0/0

### End simulateText ###
desired_targets=1
### Begin overrides ###
# User-specified persistent global and player parameters will be set here.

#### Reproduction Steps:
This error seems to occur with all characters. My last attempts have resulted in abrupt shutdowns. In the past, I had thought closing other programs seemed to help SimC finish its task, but the crashes have still been happening under these conditions.

**Comment by Collisionc** [comment]
If the computer is blue screening or abruptly restarting during the simulation AND the crashes aren't consistent, it's likely either failing memory, the cpu is not getting enough voltage, or your cpu is overheating due to a cooling system that has somehow broken.

Simc is the most stressful program you can run on a computer outside of cpu benchmarks and prime95, and can expose problems with the stability in your computer that videogames normally do not. 

If you are overclocking, disable the overclock and see if the program works.
Make sure the CPU isn't overheating by using one of the various temperature monitors, speccy, speedfan, etc. 
A guide for testing memory is here - http://www.howtogeek.com/260813/how-to-test-your-computers-ram-for-problems/

**Comment by Traker2** [comment]
Hello and thank you. What happens is my screen goes black and the PC simple powers off.  I do not get a blue screen or any real warning it's going to happen. I have not seen any high cpu temp warnings going off during the sim, though I could try another while actively monitoring my cpu temps. I have considered whether I am possibly running out of RAM, I use 8gb which these days seem to not be enough anymore. 
I am not currently over-clocking, and all frequencies have been reset. I have also considered the cpu not getting enough voltage as I have seen a voltage error occur in the past, before resetting my OC settings. I will look into checking the RAM for errors.

**Comment by Collisionc** [comment]
Yeah, that definitely sounds like failing memory if you've already checked temperatures/voltage. Sometimes Windows doesn't put up a blue screen when it crashes as well. 

The amount of memory is not important, as simc will just save information to the pagefile if it runs out of memory.

**Comment by Traker2** [comment]
Update:  While monitoring my temps during a sim I watched my core clocks reach 81° C, with the sim subsequently crashing. After getting into my (Asus) bios, I switched back on the Cool n Quiet funcion, which I had previously turned off for OC purposes. I also noticed my Voltage control was set to Manual, and set it to Auto. Save and boot, I've run 3 Sims so far, reaching a max of 70° C on average, without any issue. 

I also checked my RAM for errors and found none, so it seems it was user error that caused the problems. I've been trying to figure it out for a good while so it'll be nice to have everything running smoothly again if that was the cause.  

Thanks again.
